drivers/dummy/driver.go

```diff
@@ -0,0 +1,74 @@
+package dummy
+
```

Comment:
Suggestion.  It might be nice to have a top level comment here.  Perhaps something like:

The dummy driver allows the use of storage for containers but without the need of managing
images.

Reply:
added the comment and pushed a new version